src/gafaelfawr/config.py

```diff
@@ -318,6 +318,9 @@ class Settings(BaseSettings):
     error_footer: Optional[str] = None
     """HTML to add (inside ``<p>``) to login error pages."""
 
+    slack_webhook_file: Optional[str] = None
+    """File containing the Slack webhook to which to post alerts."""
```

Comment:
Is this the file contents or the file path? Ah, its the filepath, right.

Reply:
Yeah, maybe I should change all those `_file` instances to `_path`? Or `_filepath`? Happy for any advice there.